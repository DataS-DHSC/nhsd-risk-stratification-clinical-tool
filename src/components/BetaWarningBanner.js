import React from 'react';
import { ROUTE_FEEDBACK } from '../constants/routes';

const BetaWarningBanner = () => (
  <div className="top-banner">
    <div className="nhsuk-width-container" id="beta-warning-banner">
      <strong className="status-box status-box-blue">BETA</strong>
      <span>
        {' '}
        We want to hear from you. Give us{' '}
        <a
          href={`${ROUTE_FEEDBACK}`}
          target="_blank"
          rel="noreferrer noopener nofollow"
          id="beta-warning-banner-link"
        >
          feedback
        </a>{' '}
        to help us improve this service.
      </span>
    </div>
  </div>
);

export default BetaWarningBanner;
