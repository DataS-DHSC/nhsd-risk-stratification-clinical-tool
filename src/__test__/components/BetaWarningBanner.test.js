import React from 'react';
import { render } from '@testing-library/react';
import BetaWarningBanner from '../../components/BetaWarningBanner';

describe('BetaWarningBanner', () => {
  it('should match snapshot ', () => {
    expect(render(<BetaWarningBanner />).asFragment()).toMatchSnapshot();
  });
});
