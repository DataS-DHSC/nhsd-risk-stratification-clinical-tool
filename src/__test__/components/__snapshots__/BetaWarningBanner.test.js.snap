// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BetaWarningBanner should match snapshot  1`] = `
<DocumentFragment>
  <div
    class="top-banner"
  >
    <div
      class="nhsuk-width-container"
      id="beta-warning-banner"
    >
      <strong
        class="status-box status-box-blue"
      >
        BETA
      </strong>
      <span>
         We want to hear from you. Give us 
        <a
          href="/ras/page/feedback"
          id="beta-warning-banner-link"
          rel="noreferrer noopener nofollow"
          target="_blank"
        >
          feedback
        </a>
         to help us improve this service.
      </span>
    </div>
  </div>
</DocumentFragment>
`;
